modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/IgniteCacheSqlQueryErrorSelfTest.java

```diff
@@ -160,6 +160,12 @@ public void testDdlWrongColumnName() throws Exception {
 
         checkSqlErrorMessage("alter table test drop column wrong",
             "Failed to parse query. Column \"WRONG\" not found");
+
+        checkSqlErrorMessage("create table test(id integer primary key, AgE integer, AGe integer)",
+            "Duplicate column name: AGE");
+
+        checkSqlErrorMessage("create table test(\"id\" integer primary key, \"age\" integer, \"age\" integer)",
+            "Duplicate column name: age");
```

Comment:
Let's add another one check with different type of columns with the same name

Reply:
Done.